perllib/Open311.pm

```diff
@@ -260,7 +260,11 @@ sub _generate_service_request_description {
         $description .= "detail: " . $problem->detail . "\n\n";
         $description .= "url: " . $extra->{url} . "\n\n";
         $description .= "Submitted via FixMyStreet\n";
-        $description = "title: " . $problem->title . "\n\n$description";
+        if ($problem->cobrand eq 'brent' && $problem->contact->email =~ /ATAK/) {
```

Comment:
Brent's `new_report_title_field_label` is "Location of the problem", so I think you can drop the `&& $problem->contact->email =~ /ATAK/` bit as this will be useful for non-ATAK reports as well.